File: python/rna-transcription/rna_transcription.py
def to_rna(dnaStrand):
    rnaMap = {
        "G": "C",
        "C": "G",
        "T": "A",
        "A": "U"
    }
    rnaStrand = ""
    for dna in dnaStrand:
        rnaStrand += rnaMap[dna]
    return rnaStrand

File: javascript/rna-transcription/rna-transcription.js
var DnaTranscriber = function() {

  strandData = {
    "C": "G",
    "G": "C",
    "A": "U",
    "T": "A"
  };

  this.toRna = function(strand) {
    rnaStrand = "";
    for (var i = 0; i < strand.length; i++) {
      rnaStrand += strandData[strand.charAt(i)];
    }
    return rnaStrand;
  };

};

module.exports = DnaTranscriber;
